README.md

```diff
@@ -24,13 +24,25 @@ you can synchronize your files over your devices.
 * [YunoHost demo](https://demo.yunohost.org/nextcloud/)
 * [Official demo](https://demo.nextcloud.com/)
 
-## Configuration
-
 ## Documentation
 
  * Official documentation: https://docs.nextcloud.com/server/18/user_manual/
  * YunoHost documentation: https://github.com/YunoHost/doc/blob/master/app_nextcloud_fr.md
```

Comment:
We could link to the english documentation tho' https://github.com/YunoHost/doc/blob/master/app_nextcloud.md